src/functionalTests/java/uk/gov/hmcts/reform/finrem/functional/bulkscan/CcdBulkScanIntegrationTest.java

```diff
@@ -57,63 +62,72 @@ public void givenOcrPayload_whenTransformedPayloadUploadedToCcd_thenCaseIsCreate
         Map caseCreationDetails = ResourceLoader.jsonToObject(transformedOcrData.getBytes(StandardCharsets.UTF_8), Map.class);
         Object caseData = ((Map) caseCreationDetails.get("case_creation_details")).get("case_data");
 
-        try {
-            UserDetails userDetails = idamUtils.createCaseworkerUser();
-            log.info("case data = {}, user details = {}", caseData, userDetails);
-            submitCase(caseData, userDetails);
-        } finally {
-            idamUtils.deleteTestUsers();
-        }
+        UserDetails userDetails = idamUtils.createCaseworkerUser();
+        log.info("case data = {}, user details = {}", caseData, userDetails);
+        CaseDetails caseDetails = submitCase(caseData, userDetails);
+
+        Map<String, Object> persistedCaseData = caseDetails.getData();
```

Comment:
could we extract the assert to another method? I'm pretty sure it will grow with time and become a separate thing - we should probably already add all the other keys that are present in the test file (but this could be on a different ticket)

Reply:
Hi @jakub-kozlowski. You're right. It will grow. My follow-up PR is already growing that list.
When I raise it, please have a look to see if you feel it deserves a new method. If so, I'll create one then, if you don't mind.